DOCS-3602 Dimensioning Private Locations Edits (#14432)
DataDog/documentation: content/en/synthetics/troubleshooting/_index.md

{{< img src="synthetics/fix_user_locator.mp4" alt="Fixing User Locator error" video="true" width="100%" >}}

#### I am having issues with a CSS pointer property

> Automated browsers do not support emulating the CSS `pointer` media feature. Due to this, synthetics browser tests have `pointer: none` for all tests and devices (laptop/tablet/mobile).

Reviewer: ```suggestion
Automated browsers do not support emulating the CSS `pointer` media feature. Browser tests have `pointer: none` for all tests and devices (laptop, tablet, or mobile).
```
